ocs_ci/deployment/helpers/external_cluster_helpers.py

```diff
@@ -136,9 +136,9 @@ def get_external_cluster_details(self):
             params = f"{params} --run-as-user {ceph_user}"
 
         if config.EXTERNAL_MODE.get("use_rbd_namespace"):
-            rbd_namespace = config.EXTERNAL_MODE.get(
-                "rbd_namespace"
-            ) or self.create_rbd_namespace(rbd=rbd_name)
+            rbd_namespace = self.create_rbd_namespace(
+                rbd=rbd_name, namespace=config.EXTERNAL_MODE.get("rbd_namespace")
+            )
```

Comment:
this eliminates the passing custom namespace. create namespace only if user doesn't provide

Reply:
The custom namespace will work because we have it in the method `create_rbd_namespace`: 
https://github.com/red-hat-storage/ocs-ci/pull/11019/files#diff-541ab24068e6636c06379d0f0fa7ccf050e78732645dfd001573040e2430ae34R502

Reply:
that will create namespace. If user provide namespace of their own which means it already exists in RHCS

Reply:
```rbd_namespace = config.EXTERNAL_MODE.get(
                "rbd_namespace"
            ) or self.create_rbd_namespace(rbd=rbd_name)```

Reply:
``` rbd_namespace = config.EXTERNAL_MODE.get(
                "rbd_namespace"
            ) or self.create_rbd_namespace(rbd=rbd_name) ```

Reply:
Okay, got it.